data_steward/analytics/cdr_ops/combined.py

```diff
@@ -857,3 +858,21 @@ def verify_dataset_labels(dataset):
     result.append(query)
 results = '\nUNION ALL\n'.join(result)
 execute(client, results)
+
+# ## Verify no participant in the pdr_ehr_dup_report list has EHR data.
```

Comment:
To check for EHR data we need to verify that no person in the ehr_duplicate_table has ehr data in any of the clinical tables for eg.. measurement, visit_occurrence, procedure_occurrence etc. You can identify if any ehr records exist by joining a domain table and the mapping/ext table and filtering like below

```
select *
FROM
    `{{project}}.{{dataset}}.{{domain_table}}` d
  JOIN
    `{{project}}.{{dataset}}.{{mapping_domain_table}}` md
  USING
    ({{domain_table}}_id)
  WHERE
    person_id IN (
    SELECT
      person_id
    FROM
      `{{project}}.{{sandbox_dataset}}.{{unconsented_lookup}}`)
    AND src_dataset_id LIKE '%ehr%' 
  )
```